c/interface.h

```diff
@@ -0,0 +1,77 @@
+#include <math.h>  // For NAN
```

Comment:
I don't think this file needs to be committed, it should be auto-generated by `make`